test/scripts/run-test.sh

```diff
@@ -0,0 +1,22 @@
+#/usr/bin/env bash
+
+# Run a previously built test, generating in `$DIR`:
+# - a `cmd.sh` script containing the actual command used to run the test in an
+#   engine
+# - a `fs` directory which the test may have used for file IO
+# - an `output.log` file containing the output of the test
+#
+# Usage: DIR=... WASM=... ENGINE=... ./run-test.sh
+
+ENGINE="${ENGINE:-wasmtime}"
+[ -n "$WASM" ] || (echo "missing WASM variable" && exit 1)
+[ -n "$DIR" ] || (echo "missing DIR variable" && exit 1)
+
+pushd $DIR > /dev/null
+    mkdir -p fs
+    echo "$ENGINE $WASM" > cmd.sh
+    chmod +x cmd.sh
+    ./cmd.sh &> output.log
+    [ $? -eq 0 ] || echo "Test failed" >> output.log
+popd > /dev/null
+
```

Comment:
Trailing newline here.

Reply:
All of my files have that to avoid GitHub's red-line warnings.

Reply:
This one seems to have an extra one though.